code/game/machinery/floor_igniter.dm

```diff
@@ -15,6 +15,15 @@
 	/// Are we on?
 	var/on = FALSE
 
+/obj/machinery/igniter/shadow
```

Comment:
Worth adding a comment explaining where these are being used, or even move it to its own file like its done with ruins specific items.

Reply:
sure, onne moment

Reply:
done